### PR TITLE
Lesson 1.4

### PR DESCRIPTION
**Pull request by stephskas**
initial commit for lesson-1-4 move TodoList into State, control AddTodoForm input, add New Todo to List, destructure props

### CODE REVIEW

File: src/AddTodoForm.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        id: Date.now()
+      }
+    )
+
+    setTodoTitle("");
+  }
+  return (
+    <form onSubmit={(event) => handleAddTodo(event)}>
+      <label htmlFor="todoTitle">Title </label>
+      <input type="text" name="title" value={todoTitle}onChange={(event) => handleTitleChange(event)} />
```
Comment (royemosby) [inline src/AddTodoForm.js:24]: nit: add space between `value={todoTitle}` and `onChange={(ev...`

Comment (royemosby) [reply src/AddTodoForm.js:24]: btw, nits are what I call small changes or tweaks

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+import AddTodoForm from './AddTodoForm';
+import TodoList from './TodoList';
+
+
+function App() {
+  const [newTodo, setNewTodo] = useState("");
```
Comment (royemosby) [inline src/App.js:7]: Don't forget to clean up unused code.

File: src/TodoList.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import TodoListItem from "./TodoListItem";
+
+function TodoList({ todoList }){
```
Comment (royemosby) [inline src/TodoList.js:4]: Perfect use of destructuring!

File: src/TodoListItem.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+function TodoListItem({ todo }) {
+  return (
+    <li key={todo.key}>{todo.title}</li>
```
Comment (royemosby) [inline src/TodoListItem.js:5]: Same here as earlier PR: key is already used in TodoList.js. Remove here.

**Review by royemosby — APPROVED** [review]
Good work! Code works as intended. Please incorporate the suggested changes in your next PR- no need to update this one.

File: src/AddTodoForm.js (hunk 1 above)
Comment (stephskas) [reply src/AddTodoForm.js:24]: Thanks for highlighting that. Looks like my Prettier extension isn't working. Is there a code formatter you recommend? It looks like spacing was corrected on lesson-1-6 and it isn't an issue on current branch.

**Pull request merged**